addon/mixins/flexberry-maplayer-actions-handler.js

```diff
@@ -206,6 +206,9 @@ export default Ember.Mixin.create({
       }
 
       childLayers.pushObject(childLayer);
+
+      let rootLayer = this.get('model.rootLayer');
```

Comment:
It won't work if there are several maps on the page & they are stored not in model, but somewhere else.
So we should walk up the hierarchy from child layer till root.